lib/working_hours/config.rb

```diff
@@ -1,7 +1,35 @@
 require 'set'
 
 module WorkingHours
-  InvalidConfiguration = Class.new StandardError
+  class InvalidConfiguration < StandardError
+    attr_reader :data, :error_code
+
+    def initialize(error_code, data: {})
+      @data = data unless data.blank?
```

Comment:
Quick comment here: not sure why we default data to `{}` if we don't store it when blank? why not default data to `nil` in this case (and remove the `unless`)?

Reply:
Oh yeah good idea, it's indeed simpler